packages/typescript/src/preflight.ts

```diff
@@ -0,0 +1,17 @@
+import { PluginContext } from 'rollup';
+import { ModuleKind } from 'typescript';
+
+import { TypeScriptConfig } from './options/tsconfig';
+
+const moduleError = `
+Rollup requires that TypeScript produces ES Modules. Unfortunately your configuration specifies a
+ "module" other than "esnext". Unless you know what you're doing, please change "module" to "esnext"
+  in the target tsconfig.json file or plugin options.`.replace(/\n/g, '');
```

Comment:
the start of this line is becoming a double space

Reply:
```suggestion
 in the target tsconfig.json file or plugin options.`.replace(/\n/g, '');
```